packages/mobile/public/index.html

```diff
@@ -7,9 +7,9 @@
         <title>IOTA desktop app</title>
 
         <link rel="icon" type="image/png" href="/favicon.png" />
-        <link rel="stylesheet" href="/build/bundle.css" />
+        <link rel="stylesheet" href="/build/index.css" />
 
-        <script defer src="/build/bundle.js"></script>
+        <script defer src="/build/index.js"></script>
```

Comment:
`<script type="text/javascript" defer src="/build/index.js"></script>`

Reply:
Same here, WHATWG specs says: authors to should omit the type attribute instead of redundantly setting it.
https://html.spec.whatwg.org/multipage/scripting.html#attr-script-type

Also text/javascript it's obsolete nowdays, correct one would be application/javascript
https://www.iana.org/assignments/media-types/text/javascript